### PR TITLE
Stress tests for startup and scenarion load

### PR DESCRIPTION
**Pull request by markdomeng**
Loop 100 times in automated startup and scenario loading

### CODE REVIEW

File: MT_Test_Startup.ahk [hunk 1]
```diff
@@ -0,0 +1,100 @@
+﻿#Include <definitions>
```
Comment (markdomeng) [inline MT_Test_Startup.ahk:1]: Issue #7 

Tested on my Local pc. Not yet tested in CI. logs are attached below.
[MT_Local_Test_Startup_Logs.TXT](https://github.com/NWLdragonite/MT_Automator/files/43589/MT_Local_Test_Startup_Logs.TXT)

**Comment by francis-bit** [comment]
Transferred task to @markdomeng since he already finished it

File: MT_Test_ScenarioLoad.ahk [hunk 2]
```diff
@@ -0,0 +1,110 @@
+﻿#Include <definitions>
```
Comment (markdomeng) [inline MT_Test_ScenarioLoad.ahk:1]: Issue #13 
Tested on my Local pc. Not yet tested in CI. logs are attached below.

[MT_Local_Test_ScenarioLoad_Logs.txt](https://github.com/NWLdragonite/MT_Automator/files/43593/MT_Local_Test_ScenarioLoad_Logs.txt)

**Comment by mancajas** [comment]
:thumbsup:

File: MT_Test_ScenarioLoad.ahk (hunk 2 above)
Comment (reignerjoy) [inline MT_Test_ScenarioLoad.ahk:1]: :+1:

**Pull request merged**